OWNERS

```diff
@@ -9,7 +9,6 @@
         {name: 'cramforce', requestReviews: false},
         {name: 'dvoytenko', requestReviews: false},
         {name: 'jridgewell', requestReviews: false},
-        {name: 'rsimha', requestReviews: false},
       ],
```

Comment:
Should probably add `newmuis` and/or me here?

Reply:
SGTM, but let's do this in a separate PR for tracking.

Reply:
This should be done in the same PR, because we'd have one less global approver to make this change otherwise

Reply:
Adding a new repo-wide owner should be a separate action in its own PR created by the new owner and approved by existing owners. Piggybacking on a clean up PR makes it difficult to track. Admins can do this without the need for an extra global approver.